### PR TITLE
Added segments from phonetic/phonemic transcriptions.

### PR DESCRIPTION
**Pull request by tresoldi**
This commit adds code, partly using material from @LinguList previous version, to extract segments for entries marked as either Phonetic or Phonemic transcription. Results in `cldf/` are also updated.

This generates an initial output for transcriptions, approximate and pragmatic, listing problems that, in large part, can be later solved with an appropriate orthographic profile (the largest source of errors is the fact that some transcriptions marked as phonetic actually use (X-)SAMPA; there are also traditional problems such as look-alikes and custom notations such as "š").

Results can be further improved by mapping entries only given in Cyrillic (i.e., not alternative transcription) to the standard phonetic values for each letter in Russian. This can also be accomplished with a profile, and in fact most of the letters are already in the transcription errors (as some entries are marked as phonemic transcriptions but are actually using the Cyrillic one).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,22 +14,24 @@ Conceptlist in Concepticon: [Key-2016-1310](http://concepticon.clld.org/contribu
...
-![Glottolog: 100%](https://img.shields.io/badge/Glottolog-100%25-brightgreen.svg "Glottolog: 100%")
+![Glottolog: 98%](https://img.shields.io/badge/Glottolog-98%25-green.svg "Glottolog: 98%")
 ![Concepticon: 100%](https://img.shields.io/badge/Concepticon-100%25-brightgreen.svg "Concepticon: 100%")
-![Source: 22%](https://img.shields.io/badge/Source-22%25-red.svg "Source: 22%")
+![Source: 20%](https://img.shields.io/badge/Source-20%25-red.svg "Source: 20%")
+![BIPA: 45%](https://img.shields.io/badge/BIPA-45%25-red.svg "BIPA: 45%")
+![CLTS SoundClass: 44%](https://img.shields.io/badge/CLTS%20SoundClass-44%25-red.svg "CLTS SoundClass: 44%")
 
-- **Varieties:** 329
+- **Varieties:** 296
```
Comment (xrotwang) [inline README.md]: Do you really want to remove 33 wordlists from the dataset?

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tresoldi) [reply README.md]: I certainly don't! I'll check what happened, sorry.

Comment (LinguList) [reply README.md]: The problem is that the form stripping function returns none if the transcription is not in a certain range. So we are missing one other type of transcription apart from latin, cyrillic, or whatsoever here, and this kills the thirty languages. Should be easy to fix by looking into the different transcriptionvalues.

Comment (tresoldi) [reply README.md]: That's indeed the problem, I'm looking at the output of the previous `clean_form()` in order to look for the best solution. Unfortunately, given the variety of sources/languages and usages of stuff like brackets, the less worst one is likely to keep everything as it was, in order not to lose any valid information.

Comment (LinguList) [reply README.md]: Yes, the form should be kept as is, if it's not '???' or similar. I think, ther'es simply one more type of trnascription not accounted for.

Comment (tresoldi) [reply README.md]: I am cleaning most forms, as in your code, but defaulting to keeping the transcription in the worst cases. The current filter is good enough and there is no point in working through the data before having at least a preliminary orthographic profile (which might solve most of the problems).

**Comment by tresoldi** [comment]
I also realized that one more subtle change is in the installed version of glottolog and concepticon reported in the metadata:

```
"glottolog_version": "pyglottolog-1.2.1-60-gf46bb64",
"concepticon_version": "pyconcepticon-1.4.0-39-g121e0c7"
``` 

Do we have/need any guideline on only generating the CLDF files from releases of glottolog and concepticon published on PyPI? While in this case the generation is reproducible (as I was running everything in a virtual environment, fetching the most recent commit of each repository and installing in development mode), this could allow people to unwillingly refer to some private fork/branch. In any case, using some stable release could be considered a good practice.

It is really a minor thing, but I wanted to bring it up (btw, most metadata already in the repositories refer to some specific commit as in this case, especially [only?] for concepticon).

**Review by LinguList — APPROVED** [review]

**Pull request closed without merging**